challenges/challenge-2/challenge-2.css

```diff
@@ -375,39 +375,39 @@ body {
 	position: absolute;
 	top: 25rem;
 	left: 25rem;
-	animation: tardis_spin;
+	animation: rotateOut;
 	animation-duration: 10s;
 	animation-timing-function: linear;
 	animation-iteration-count: infinite;
 	z-index: 110;
 }
-@keyframes tardis_spin {
+@keyframes rotateOut {
```

Comment:
@DavidMatthewFraser Are these keyframes nested correctly?

Reply:
Looks like it yes 👍